integration_tests/mockApis/incentivesApi.ts

```diff
@@ -16,4 +16,208 @@ export default {
       },
     })
   },
+  stubGetAvailableLevels: (): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'GET',
+        urlPattern: '/incentivesApi/iep/levels/MDI',
+      },
+      response: {
+        status: 200,
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: [
+          {
+            iepLevel: 'BAS',
+            iepDescription: 'Basic',
+            sequence: 1,
+            default: false,
+          },
+          {
+            iepLevel: 'STD',
+            iepDescription: 'Standard',
+            sequence: 2,
+            default: true,
+          },
+          {
+            iepLevel: 'ENH',
+            iepDescription: 'Enhanced',
+            sequence: 3,
+            default: false,
+          },
+        ],
+      },
+    })
+  },
+  stubGetReviews: (): SuperAgentRequest => {
```

Comment:
As the other stub is named `stubGetIncentivesLevelStandard` for consistency you could name this one `stubGetIncentivesLevelBasic`